### ISSUE TITLE
contentFor in partials

### ISSUE THREAD

**Issue by random-char** [gobuffalo/plush#100]
Receiving error when using ```contentFor``` in partials.  Need to move them up (see example for better understanding):
The way it works
```
//entity/create.html
<h1>Create</h1>
<%= contentFor("extraJS") {%>
    var someVar = 1;
<%} %>
<%= partial("entity/form.html") %>
```
The way it doesn't (contentFor block is not present in create.html anymore)
```
//entity/_form.html
<%= form...%>
<%= contentFor("extraJS") {%>
    var someVar = 1;
<%} %>
```
Is it expected or am I doing something wrong?

**Comment by random-char** [comment]
After spending some time, I've stumbled upon an issue that the context in the partial and in outer template are different. Is there a way to use same outer context in the partial?

**Comment by paganotoni** [comment]
I've also faced this one. a PR is welcome @random-char!

**Comment by paganotoni** [comment]
@random-char this one may be fixed with #140 want to give it a shot?
